src/styleUtils.js

```diff
@@ -64,19 +63,18 @@ define(
 		);
 	};
 	
-	// Uses the Flash based downloadify library to save files to the local file system
+	// Uses the FileSaver.js library to save files to the local file system
 	var saveCsl = function (
 			cslCodeOrData,
 			styleId,
 			comment // optional, if comment is included cslCodeOrData
 			        // must be a CSLEDIT_Data instance
 			) {
 		var dialog = $('<div title="Save CSL Style">' +
```

Comment:
> also it wasn't changed by you?

No, this is the first time I touch this code.



> I would suggest to write this in one line

Agree that that's clearer. Would probably put the comment on top, though, which seems more common for describing function calls.